src/base/ctrl_socket.c

```diff
@@ -220,6 +221,8 @@ CTRL_FUNC(pause)
 		xdebug_xml_add_text(action, xdstrdup("IDE Connection Signalled"));
 
 		XG_DBG(context).do_connect_to_client = 1;
+
+		xdebug_enable_debugger_and_rebuild_stack_if_disabled();
```

Comment:
When starting the debugger from the socket, we need to enable the debugger and rebuild the stack. Notice: I have not been able to test this as I have no idea how to enable this socket functionality, but I have no reason to believe it won't work in the same way as in other jit cases